actions.yaml

```diff
@@ -7,3 +7,9 @@ set-tls-private-key:
     private-key:
       type: string
       description: The content of private key for communications with clients. Content will be auto-generated if this option is not specified.
+
+pre-upgrade-check:
+  description: Run necessary pre-upgrade checks before executing a charm upgrade.
+
+resume-upgrade:
```

Comment:
In general, pgbouncer may not need `resume-upgrade` action at all.
We have it for databases because the new version of the database can change files in the data-dir in an incompatible way.
pgbouncer most probably supports downgrade without any issues, so `resume-upgrade` maybe not needed

Reply:
I would keep it for:
* identical UX for postgresql and pgbouncer
* manual ability to upgrade postgresql and pgbouncer in parallel: 
```
juju refresh pgbouncer-k8s
juju refresh postgresql-k8s
# check biggest ordinal upgrade results
juju run pgbouncer-k8s resume-upgrade
juju wait-for-active application pgbouncer-k8s
juju run postgresql-k8s resume-upgrade
juju wait-for-active application postgresql-k8s
```